src/window_background/globals.ts

```diff
@@ -6,45 +6,45 @@ import * as mtgaLog from "./mtga-log";
 // Hopefully we'll be able to get rid of all of the ones that can change,
 // and put them into stores or better structures than a giant export list.
 
-let actionLogDir = "";
+const actionLogDir = "";
```

Comment:
`globals.actionLogDir` is a mutable binding, otherwise it'd always be `""`.
```suggestion
let actionLogDir = "";
```